### ISSUE TITLE
Graphics error after new update on Mac

### ISSUE THREAD

**Issue by infactable** [ValveSoftware/Dota2-Gameplay#11143]
### Description

This is how my game looks after new summer update, it's the same in ranked and vs bots.

Please help.

### Screenshots

<img width="2240" alt="Screenshot 2023-08-31 at 14 48 28" src="https://github.com/ValveSoftware/Dota2-Gameplay/assets/58468623/1260d47b-823c-48d4-86fe-7d057020408d">

**Comment by spasik88** [comment]
it's effect from treasury, u can off it in arsenal (somewhere in general subjects). he will disappear after 07.09.2023

**Comment by infactable** [comment]
Yes I found it in map effects, as I can see the point is to play dota but
like you are in the 90s lol. Anyway I can't turn it off and I'm used to
play on Retina 4K display, so this really annoys me, it's like playing on
GameBoy.

**Comment by infactable** [comment]
It works! Now it finally shows an option unequip. Thanks!!

**Comment by th3daywalk3r** [comment]
it's not a bug, delete it please

**Comment by Hotdog541** [comment]
fix my issue. on a mac too ![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/143739712/8d2c8c55-271c-4372-af94-29c8c65a5aa2)
